### PR TITLE
Adds the crazy hamburger as a new item the chef can make

### PR DESCRIPTION
**Pull request by TeomanTheGreat**
## About The Pull Request
Adds the crazy hamburger as a cookable item. It is made with a steak, 2 chilis, 20 units of cooking oil, 2 deathnettles, 4 cheese wedges, and 1 bun.
image

image

I got the sprite from Golgor from the Oasis Station discord. He gave me permission to use it.

## Why It's Good For The Game
Funny burger

## Changelog
:cl:Phil Smith, Golgor
add: The Crazy Hamburger recipe
imageadd: Crazy Hamburger sprite by Golgor
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
add a flare to the crafting recipe

**Comment by TeomanTheGreat** [comment]
> 
> 
> add a flare to the crafting recipe

the whole joke in the video is that the flare spontaneously appeared on the burger when it was done therefore I'm gonna stay true to the crazy hamburger lore

**Comment by ghost** [comment]
update wiki btw

**Comment by TeomanTheGreat** [comment]
> 
> 
> update wiki btw

i'ill wait until it gets merged

File: code/modules/food_and_drinks/food/snacks_burgers.dm
Comment (Archanial) [inline code/modules/food_and_drinks/food/snacks_burgers.dm]:
```suggestion
	desc = "Crazy hamburger! It is horrible!"
```

**Review by Archanial — COMMENTED** [review]
we should add even more memes into the game it's funny haha big chungus

**Comment by TeomanTheGreat** [comment]
bam thanks

**Comment by qwertyquerty** [comment]
Attribute the spriter

**Comment by TeomanTheGreat** [comment]
> 
> 
> Attribute the spriter

how would I do that? You want a screenshot of him saying it's ok for me to use it?

**Comment by Archanial** [comment]
> 
> 
> > Attribute the spriter
> 
> how would I do that? You want a screenshot of him saying it's ok for me to use it?

Add him to CL. Say something like: "sprite made by".

**Comment by TeomanTheGreat** [comment]
> 
> 
> > > Attribute the spriter
> > 
> > 
> > how would I do that? You want a screenshot of him saying it's ok for me to use it?
> 
> Add him to CL. Say something like: "sprite made by".

the changelog is now updated

**Comment by Crossedfall** [comment]
Reached out the author for verification.

**Comment by TeomanTheGreat** [comment]
image

**Comment by Crossedfall** [comment]
Still reaching out to get first hand approval and so that we can properly attribute them if they want it.

**Comment by Crossedfall** [comment]
All good. Added a commit attributing the author as well. 👍

**Comment by TeomanTheGreat** [comment]
So when will this be approved?

**Comment by hotimep** [comment]
crazy

**Comment by OceanFish1** [comment]
Finally
Cumburger

**Review by qwertyquerty — APPROVED** [review]

**Comment by TeomanTheGreat** [comment]
Merge when

**Comment by TeomanTheGreat** [comment]
my god

**Pull request merged**
